NextCenturyCorporation/MCS
# Mcs 1723 doc publish requirements

doc/ had a requirements.txt that mirrored the requirements.txt at the root of MCS.  Removing docs/requirements.txt and set action to use the mcs/requirements.txt.  Tested on github and is working as expected.

To Test:
1. Tag the branch: `git tag doc-publish-test`
2. Push the tag: `git push origin doc-publish-test`
3. Create a release from this tag (`doc-publish-test`)
  1. create the release as `Set as a pre-release` (https://github.com/NextCenturyCorporation/MCS/releases)
  2. Then go to actions(https://github.com/NextCenturyCorporation/MCS/actions)
    1. The doc-publish task should pass
    2. The pypi should fail (version has already been published) 
4. After confirming that the doc-publish task has passed, delete the test release (https://github.com/NextCenturyCorporation/MCS/releases)
